fendermint/vm/actor_interface/src/ipc.rs

```diff
@@ -5,14 +5,70 @@
 // Here we define stable IDs for them, so we can deploy the
 // Solidity contracts during genesis.
 
+use fendermint_vm_ipc_actors as ia;
+pub use fendermint_vm_ipc_actors::gateway_manager_facet::SubnetID;
+use lazy_static::lazy_static;
+
+use crate::diamond::{EthContract, EthContractMap, EthFacet};
+
 define_id!(GATEWAY { id: 64 });
 define_id!(SUBNETREGISTRY { id: 65 });
 
+lazy_static! {
+    pub static ref IPC_CONTRACTS: EthContractMap = {
+        [
+            (
+                "GatewayDiamond",
+                EthContract {
+                    actor_id: GATEWAY_ACTOR_ID,
+                    abi: ia::gateway_diamond::GATEWAYDIAMOND_ABI.to_owned(),
```

Comment:
@aakoshh, just a quick note, @dnkolegov was working on [an additional facet](https://github.com/consensus-shipyard/ipc-solidity-actors/pull/171/files) to the gateway to reduce even more the size of the contract. I will review the PR next, I don't know if you want to include it already in this PR or have a follow-up once is merged.

Reply:
I would merge this as it is and open a new PR. That PR will make it much easier to see what are the typical changes needed when something like this happens.